sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -919,12 +919,6 @@ qualifiedNameList
     : qualifiedName (',' qualifiedName)*
     ;
 
-functionName
```

Comment:
this is for ANSI-compliant, not pgsql dialect, we should keep it.

Reply:
ditto